From cb36e7c6d5e7fd3220c33b658934296e51fd0f4d Mon Sep 17 00:00:00 2001
From: Kha Tran <kha.tran.px@rvc.renesas.com>
Date: Wed, 24 Jan 2018 17:53:28 +0700
Subject: [PATCH 348/628] ARM: DTS: r8a7744: Add description for PWM

Add the seven PWM instances found in the r8a7744 to DT with a disabled
status.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7744.dtsi | 56 ++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 56 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 04d2ff9..98eb6dd 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1253,6 +1253,62 @@
 		status = "disabled";
 	};
 
+	pwm0: pwm@e6e30000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e30000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm1: pwm@e6e31000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e31000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm2: pwm@e6e32000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e32000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm3: pwm@e6e33000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e33000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm4: pwm@e6e34000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e34000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm5: pwm@e6e35000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e35000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
+	pwm6: pwm@e6e36000 {
+		compatible = "renesas,pwm-rcar";
+		reg = <0 0xe6e36000 0 0x8>;
+		clocks = <&mstp5_clks R8A7744_CLK_PWM>;
+		status = "disabled";
+		#pwm-cells = <2>;
+	};
+
 	ipmmu_sy0: mmu@e6280000 {
 		compatible = "renesas,ipmmu-vmsa";
 		reg = <0 0xe6280000 0 0x1000>;
-- 
2.7.4

